src/cc_catalog_airflow/dags/popularity_workflow.py

```diff
@@ -0,0 +1,88 @@
+import os
+import logging as log
+from datetime import datetime
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from util.popularity.sql import (
+    upload_normalized_popularity, build_popularity_dump_query,
+    dump_selection_to_tsv
+)
+from util.popularity.math import (
+    generate_popularity_tsv, get_percentiles
+)
+from util.operator_util import get_log_operator
+"""
+Dump all of the popularity data to the disk, compute the popularity score for
+each row, and load it back into the database.
+
+See `util.popularity.math` for a more complete explanation of how popularity is
+calculated.
+"""
+
+DAG_ID = "popularity_workflow_dag"
+DAG_DEFAULT_ARGS = {
+    'owner': 'data-eng-admin',
+    'depends_on_past': False,
+    'start_date': datetime(2020, 1, 1),
```

Comment:
The start date for Apache Airflow DAGs is usually either:
- Some arbitrary date in the past, or
- A date in the past chosen to allow 'backfilling', which is running the dag from the 'perspective of' a past date.  (the `start_date` defines the first date for which this perspective should exist).

It has to be static, i.e., using something dynamic like `datetime.datetime.now()` would cause inconsistencies in the Airflow DB.